data/fieldlist_GFDL.jsonc

```diff
@@ -36,13 +36,13 @@
     "ucomp": {
       "standard_name": "eastward_wind",
       "units": "m s-1",
-      "scalar_coord_templates": {"plev": "ucomp{value}"},
+      "scalar_coord_templates": {"plev": "u{value}"},
```

Comment:
Will these changes be backwards-compatible?

I wonder if the notion of a singe GFDL field list will hamstring supporting multiple model configurations.

Reply:
It's not backwards compatible, but as far as I know only in the positive, "fixing a bug" way. The example prompting this change came from Raf's high-res run: atmos/ts/monthly provides ucomp and vcomp, but /daily has u500, etc. I've never seen u at 500 mb referred to as `ucomp500`, which is what the previous entry in the code specified. This change means u500 et al. will now be properly detected, without breaking anything (since 'ucomp500' et al. don't exist in the wild).

In terms of your second point, we could create GFDL_2, GFDL_3, ... conventions as needed, but the larger issue that you (and issue #257) are getting at is that we need to be more granular and flexible with our naming conventions. Allowing multiple conventions per experiment (data source) is one way to address it, but I'm wondering if it might be worth going farther and allowing multiple name possibilities at the level of individual variables (e.g. "'eastward_wind' might be called '`u`', or '`ua`', or '`ucomp`', or...")

Reply:
Github is marking this PR as merged even though I didn't merge it directly into /develop, because I merged it into #161. To ensure this question doesn't get buried, I moved it to discussion #260.